src/test/js_dictionary.cpp

```diff
@@ -0,0 +1,11 @@
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+#include <vector>
+#include "js_dictionary.hpp"
+
+using Catch::Matchers::Contains;
+
+TEST_CASE( "Testing DictionarySchema#Constructor" ) {
+    realm::js::DictionarySchema schm {"string{}"};
```

Comment:
[Questions]  Is the postfix for `dictionary` not supposed to be `<>`?  I'm using `{}` for `set` at the moment.

Reply:
Our dictionaries look very much like plain JavaScript objects (key is a string). For example `{ name: "Arthur", age: 42 }` - so maybe `{}` is a better fit for dictionaries.